### PR TITLE
Implement new styling of the footer

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
https://trello.com/c/wjLFraLX/57-dhdid%3A-implement-desktop-css-for-footer

Its time that the footer had some design love. This is being implemented for the new desktop header/footer (behind a test) and for the old footer. 

TODO in this PR:
- [x] Browserstack testing

TODO in seperate PR:
- AMP footer

## What is the value of this and can you measure success?
Looks nicer!

## Does this affect other platforms - Amp, Apps, etc?
No, but the AMP footer needs some love too (added it to the todo in the next PR).

### Mobile
image

### Desktop (in test)
image

### Desktop (Current)
image

## Tested in CODE?
Nope

### CODE REVIEW

**Comment by NataliaLKB** [comment]
Unfortunately I ran into the css issue on AMP (css too big). Its probably better to tackle that issue in a separate PR which I will do once this is merged. Can I have a review @SiAdcock @gustavpursche ?

File: common/app/views/fragments/footer.scala.html [hunk 1]
```diff
@@ -76,211 +76,298 @@
                     </div>
                 }
 
-            <ul class="colophon__list">
+            <div class="colophon__lists-container">
```
Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: None of the links changed, but I moved the markup around quite a lot. Sorry for confusing diffs 😞

File: static/src/stylesheets/module/email-signup/_form.scss [hunk 2]
```diff
@@ -193,27 +193,47 @@
...
+
+        &:hover {
+            background-color: darken($neutral-2, 15%);
+            border-color: darken($neutral-2, 15%);
+        }
+
+        &:active,
+        &:focus {
+            background-color: #ffffff;
+            border-color: #ffffff;
```
Comment (gtrufitt) [inline static/src/stylesheets/module/email-signup/_form.scss]: Is the text still visible with a white background?

**Comment by gtrufitt** [comment]
I remember this being very fun to get right between breakpoints, looking at the desktop and tablet breakpoints, it would seem that the email sign-up would end up in a very wide box - is this your intention?

**Comment by gtrufitt** [comment]
Love the new black style email sign-up  💯

File: static/src/stylesheets/layout/_footer.scss [hunk 3]
```diff
@@ -70,96 +70,116 @@ $c-primary-footer-background-side-bar: mix($c-primary-footer-background, #ffffff
...
+        display: inline-block;
+        padding-bottom: $gs-baseline;
+        text-transform: lowercase;
     }
 
-    @include mq(desktop) {
-        @include columns(3, $gs-gutter);
-        padding-left: $gs-gutter;
-    }
+    &:before {
```
Comment (gustavpursche) [inline static/src/stylesheets/layout/_footer.scss]: Is it not possible to express this as a border?

File: static/src/stylesheets/module/email-signup/_form.scss (hunk 2 above)
Comment (NataliaLKB) [inline static/src/stylesheets/module/email-signup/_form.scss]: Yep! 
image

Comment (gtrufitt) [inline static/src/stylesheets/module/email-signup/_form.scss]: Ah yeah, sorry, thought it was applied to the submit, my bad! :)

**Comment by NataliaLKB** [comment]
@gtrufitt 
> I remember this being very fun to get right between breakpoints, looking at the desktop and tablet breakpoints, it would seem that the email sign-up would end up in a very wide box - is this your intention?

Tablet size ends up like this: 
image
 with a [max-width](https://github.com/guardian/frontend/pull/16986/files#diff-1f24cbf4024897e9a6ee3955dad3e3feR199). But I wonder if the max-width should be smaller. Thoughts? 

Here is the gif resizing:
new-footer-resizing

**Comment by gtrufitt** [comment]
@NataliaLKB It's worth noting that the logged in user gets this:

image

Which looks odd there, but might look even odder in the wider box! Not sure how much you want to worry about it but thought I'd raise it considering the back and forth @superfrank and I did on this previously!

Personally I don't mind the max width but @superfrank did a load of work on a new footer design that never got implemented so he may be able to step in and help!

**Comment by NataliaLKB** [comment]
Thanks @gtrufitt ! Its a good thing to raise. I am happy to follow up with @superfrank to take a look at this after. I am not sure if @zeftilldeath knew about that state when he designed this and is now off for two weeks.

**Comment by NataliaLKB** [comment]
@gtrufitt @superfrank the signed in version looks like this on tablet:

image

**Comment by NataliaLKB** [comment]
@gtrufitt @gustavpursche is there anything else you would like me to change here?

**Comment by superfrank** [comment]
You can only really fix that space issue by doing an audit of the footer and deciding what really needs to be there/what doesn't as yeh it's a little awkward that email has so much space. 

However, I'd say that's fine as a stop gap until @zeftilldeath gets back and has a proper chance to consider everything.

Not sure these really help in this situation, but these were some initial designs I did for the footer:

footer-desktop
footer-mobile

**Comment by NataliaLKB** [comment]
> You can only really fix that space issue by doing an audit of the footer and deciding what really needs to be there

We started this and decided it was too much work for lack of benefit in this area (so few users actually see the footer, let alone engage with it)

**Comment by gustavpursche** [comment]
@NataliaLKB Nothing more from my side. Looks good to 🚢.

**Pull request merged**